sylt-parser/src/expression.rs

```diff
@@ -744,6 +744,13 @@ fn set_or_dict<'t>(ctx: Context<'t>) -> ParseResult<'t, Expression> {
                     exprs.push(expr);
                 }
 
+                if !matches!(ctx.token(), T::Comma | T::RightBrace) {
+                    raise_syntax_error!(
+                        ctx,
+                        "Expected an element delimiter ',' but got {:?}",
```

Comment:
```suggestion
                        "Expected a comma ',' but got {:?}",
```
It is a bit clunky to read. Is there a good reason for writing "element delimiter"?